L1Trigger/L1CaloTrigger/plugins/Phase1L1TJetProducer.cc

```diff
@@ -351,9 +368,11 @@ template <class Container>
 void Phase1L1TJetProducer::_fillCaloGrid(TH2F & caloGrid, const Container & triggerPrimitives)
 {
   //Filling the calo grid with the primitives
-  for (auto primitiveIterator = triggerPrimitives.begin(); primitiveIterator != triggerPrimitives.end(); primitiveIterator++) 
-  {
-    caloGrid.Fill((float) primitiveIterator -> eta(), (float) primitiveIterator -> phi(), (float) primitiveIterator -> pt());
+  for (auto primitiveIterator = triggerPrimitives.begin(); primitiveIterator != triggerPrimitives.end(); primitiveIterator++){
```

Comment:
There is a double curly bracket layer here. To be consistent with the coding style used in other parts of the code, I would remove the curly bracket at the end of line 371 and at line 375 and then reduce by a tabulation level.

Reply:
https://github.com/cms-l1t-offline/cmssw/pull/803/commits/a9d12f59c95bf20212263d403bbf6b794f54be08